### ISSUE TITLE
IPv6 supported?

### ISSUE THREAD

**Issue by vladns** [lbulej/htb.init#5]
Actually a question.

**Comment by lbulej** [comment]
No, even though I recall having seen a version floating around that added support for IPv6 together with ESFQ queuing discipline. I just can't find it anymore.

**Comment by vladns** [comment]
Here for example:
https://gitweb.gentoo.org/repo/gentoo.git/tree/net-misc/htbinit/files

**Comment by lbulej** [comment]
Thanks for the link! I'll see if I can eventually integrate them.
